docker/gateway-compose.yml

```diff
@@ -6,10 +6,11 @@ services:
       postgres:
         condition: service_healthy
     healthcheck:
-      interval: 10s
+      test: ["CMD-SHELL", "/usr/lib/trino/bin/health-check || exit 1"]
```

Comment:
Added healthcheck here @rdsarvar

Reply:
By default it’s using the health check defined in the Dockerfile, is your preference to be explicit?